libs/ui/search/src/lib/record-table/record-table.component.ts

```diff
@@ -14,12 +14,13 @@ import { SortByField } from '@geonetwork-ui/common/domain/search'
   styleUrls: ['./record-table.component.css'],
 })
 export class RecordTableComponent {
-  selectedRecords: string[] = []
-
+  @Input() selectedRecords: string[] = []
   @Input() records: any[] = []
   @Input() totalHits?: number
   @Input() sortBy?: SortByField
   @Output() recordSelect = new EventEmitter<CatalogRecord>()
+  @Output() recordsSelection = new EventEmitter<CatalogRecord[]>()
+  @Output() recordsDeselection = new EventEmitter<CatalogRecord[]>()
```

Comment:
I feel like the names of these events will bring confusion. How about:
* `recordClick`
* `recordsSelect`
* `recordsDeselect`?